doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -264,6 +358,19 @@ An example of a proof using cumulativity
 
    Inductive eq@{i} {A : Type@{i}} (x : A) : A -> Type@{i} := eq_refl : eq x x.
 
+.. coqtop:: none
+
+   Set Printing Universes.
+
+.. coqtop:: out
+
+   Print eq.
+
+The universe of :g:`eq` is irrelevant here, hence proofs of equalities can be
+seen as living in any other universe.
```

Comment:
"can be seen" is imprecise, how about "are"?
"as living" is also imprecise, would "exist" or "are present" be accurate?

Reply:
PS Perhaps "any other universe" -> "all other universes" as well?

Reply:
> "can be seen" is imprecise, how about "are"?
> "as living" is also imprecise, would "exist" or "are present" be accurate

No.  The "can" is important, and conveys "it is consistent to assume", not "it is actually the case that".  "Exist" and "are present" are less accurate, even if more precise; the more precise but still accurate version is something like "the universe in which the type of eq is an inhabitant".  I don't see why "living" is imprecise, since "inhabiting" is almost a(n English) synonym and is actually technical jargon.

> PS Perhaps "any other universe" -> "all other universes" as well?

No, the point is that you have to pick a single universe, but any choice will do.

Reply:
(I don't see what's wrong with the current wording, hence cannot suggest any better wording.)

Reply:
How about "proofs of equalities may inhabit any other universe."?  Good, meh, fail?

"other" is essential here?

Reply:
Sounds good, and "other" is not essential and dropping it is, I believe, a strict improvement